server/asset/dcr/dcr.go

```diff
@@ -284,16 +284,81 @@ func (dcr *Backend) VerifyUnspentCoin(coinID []byte) error {
 	return nil
 }
 
-// UnspentCoinDetails gets the recipient address, value, and confirmations of
-// unspent coins. For DCR, this corresponds to a UTXO. If the utxo does not
-// exist or has a pubkey script of the wrong type, an error will be returned.
-func (dcr *Backend) UnspentCoinDetails(coinID []byte) (addr string, val uint64, confs int64, err error) {
+// FeeCoin gets the recipient address, value, and confirmations of a transaction
+// output encoded by the given coinID. A non-nil error is returned if the
+// output's pubkey script is not a non-stake P2PKH requiring a single
+// ECDSA-secp256k1 signature.
+func (dcr *Backend) FeeCoin(coinID []byte) (addr string, val uint64, confs int64, err error) {
```

Comment:
Given the restrictions already placed on the pkScript, both CoinDetails and UnspentCoinDetails were too general-purpose sounding.

This could even take address and value to check those in the function, but i did not want to rewrite `(*AuthManager).validateFee` at this time.